CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release Notes
 
-- 2.0.0 (2021-07-XX)
+- 2.0.1 (2021-11-11)
+  - Bugfix: BLEU.corpus_score() now using max_ngram_order
+
+- 2.0.0 (2021-07-18)
```

Comment:
your master was not synced probably as this file had already some changes listed for v2.0.1.

Reply:
Fixed, thanks.